client/client_docker_test.go

```diff
@@ -50,63 +50,6 @@ func TestDockerClientConnectShutdown(t *testing.T) {
 	client.Wait()
 }
 
-func TestDockerClientBlockingSendReceive(t *testing.T) {
```

Comment:
where did all these tests go? are they no longer relevant?

Reply:
Yeah it seems we were testing some client features that catshadow doesn't even use and some of these tests stopped working at some point due to client bugs.

Reply:
@mixmasala what's your take?
My thinking is:
- If something doesn't work on `main`, and doesn't work on the PR branch, we shouldn't have to delete the tests to get an unrelated PR merged.
- If they do work on `main`, but not on this branch (due to unrelated bugs), we should open an issue and reference it as part of the PR thread, but not necessarily fix them.
- If they do not work on `main`, but do work on the PR branch, that sounds like we fixed something, or that we are lacking test coverage. If it's the latter, open an issue.
- If the code became irrelevant (no other callers), and we want to delete the tests, we should probably delete the exposed API code they exercise, too.

Reply:
I mean... I've said I have plans to fix the client library. Can't do everything in this PR.